nvidia-bert/docker/Dockerfile

```diff
@@ -1,19 +1,24 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+FROM nvidia/cuda:10.1-cudnn7-devel
 
-FROM mcr.microsoft.com/azureml/base-gpu:openmpi3.1.2-cuda10.1-cudnn7-ubuntu18.04
-RUN apt update -y && pip install --upgrade pip
+RUN apt-get update &&\
+    apt-get install -y sudo git bash
+
+RUN apt-get update && apt-get install -y --no-install-recommends &&\
+    wget zip ca-certificates build-essential curl libcurl4-openssl-dev &&\
```

Comment:
I get:

> /bin/sh: 1: wget: not found
The command '/bin/sh -c apt-get update && apt-get install -y --no-install-recommends &&    wget zip ca-certificates build-essential curl libcurl4-openssl-dev &&    libssl-dev python3-dev python3.6 python3-pip' returned a non-zero code: 127

Reply:
fixed the syntax issue. can you please retry?